report-viewer/src/App.vue

```diff
@@ -25,7 +34,47 @@ import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
 import { library } from '@fortawesome/fontawesome-svg-core'
 import { faMoon, faSun } from '@fortawesome/free-solid-svg-icons'
 import { store } from './stores/store'
+import ToastComponent from './components/ToastComponent.vue'
+import { Version, reportViewerVersion } from './model/Version'
+import { computed, ref } from 'vue'
 
 library.add(faMoon)
 library.add(faSun)
+
+const newestVersion = ref(new Version(-1, -1, -1))
```

Comment:
Since the value of the version gets determined asynchronously, and it thus changes during the runtime, there is no good place to have it, other than the place where it is needed

Reply:
The only option would be to have a file for this singular toast and use it here instead